wallet/src/account/output_cache/mod.rs

```diff
@@ -437,6 +424,36 @@ impl OutputCache {
     }
 }
 
+// Check if the TxOutput's timelock is in a specified state Locked/Unlocked or both
```

Comment:
///

Reply:
I'm trying to understand this function. Is it better to describe this function as "checks the output against the current block height and compares it with the `locked_state` parameter. If they match, the function return true, if they don't, it returns false. For example, if we would like to check that an output is locked, we pass `locked_state = WithLocked::Locked`, and pass the output in question. If the output is locked, the function returns true. Otherwise, it returns false".

If this better describes this function, please use it as documentation.